argo-cd-apps/base/toolchain-member/toolchain-member-operator.yaml

```diff
@@ -30,12 +30,6 @@ spec:
       destination:
         namespace: toolchain-member-operator
         server: '{{server}}'
-      syncPolicy:
-        automated:
-          prune: true
-          selfHeal: true
-        syncOptions:
-          - CreateNamespace=false
         retry:
```

Comment:
This looks like it needs to be removed as well.

Reply:
Done